docs/source/developer/threading_implementation.rst

```diff
@@ -29,14 +29,14 @@ The relevant source files referenced in this document are
   schedulers. Note that the basic thread local variable logic is duplicated in
   each of these files, and not shared between them.
 
-- ``numba/npyufunc/parallel.py``
+- ``numba/np/ufunc/parallel.py``
 
   This file contains the Python and JIT compatible wrappers for
   ``set_num_threads()``, ``get_num_threads()``, and ``get_thread_id()``, as
   well as the code that loads the above libraries into Python and launches the
   threadpool.
 
-- ``numba/npyufunc/parfor.py``
+- ``numba/np/ufunc/parfor.py``
```

Comment:
This is now `numba/parfors/parfor_lowering.py`

Reply:
done